orttraining/orttraining/test/python/onnxruntime_test_postprocess.py

```diff
@@ -34,7 +34,7 @@ def get_onnx_model(self, model, model_desc, inputs, device,
                            device,
                            world_rank=0,
                            world_size=1,
-                           _opset_version=12,
+                           _opset_version=14,
```

Comment:
May we remove this line so that it uses the default (whatever is defined). Else, we might miss catching opset version regressions.

Reply:
The default is 14 for ORTModule but this is an explicit parameter given to ORTTrainer.